reference/constraints/Bic.rst

```diff
@@ -73,8 +73,6 @@ will contain a Business Identifier Code (BIC).
 
         class Transaction
         {
-            protected $businessIdentifierCode;
```

Comment:
@OskarStark Wouldn't it be less confusing to always add the property instead?

Reply:
Ofc we can, but most of them does *not* have them.

You get my 👍

Reply:
Ofc we can, but most of them does *not* have them.

You get my 👍